src/provider/model/FinEnvironment.ts

```diff
@@ -28,6 +28,18 @@ interface KnownEntity {
     entityType: EntityType;
 }
 
+/**
+ * Checks if an entity is of a "page-based" entity type. These are entities that are built using a
+ * webpage/browser/DOM/etc. This only includes entity types that both meet this definition, AND are supported by
+ * the service.
+ *
+ * These entity types have a more finely controlled handshake process, as the fin API provides better eventing
+ * and querying of these entity types.
+ */
+function isPagedEntity(entityType: EntityType): boolean {
```

Comment:
Think this more naturally belongs at the bottom, being an implementation detail of the class